kubernetes/assets/dashboards/kubernetes_nodes.json

```diff
@@ -0,0 +1,1401 @@
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Allocatable memory per node",
+            "type": "timeseries",
+            "width": 57,
+            "x": 0,
+            "y": 173
+        },
+        {
+            "height": 14,
+            "legend": false,
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "q": "sum:kubernetes.containers.running{$cluster,$host} by {host}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Sum of containers per node",
+            "type": "timeseries",
+            "width": 57,
+            "x": 58,
+            "y": 190
+        },
+        {
+            "height": 13,
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "limit": 25,
+                        "order": "asc",
+                        "q": "avg:kubernetes_state.node.cpu_allocatable{$host, $cluster} by {host}"
+                    }
+                ],
+                "viz": "query_table"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Allocatable CPU sort ASC",
+            "type": "query_table",
+            "width": 57,
+            "x": 0,
+            "y": 270
+        },
+        {
+            "height": 13,
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "limit": 25,
+                        "order": "asc",
+                        "q": "avg:kubernetes_state.node.pods_allocatable{$cluster,$host} by {host}"
```

Comment:
For the query tables, is it possible to use the metric alias for the right column with something easier to understand?

For example:
<img width="635" alt="Screen Shot 2020-07-09 at 10 46 47 AM" src="https://user-images.githubusercontent.com/15065007/87054761-801e0c80-c1d1-11ea-9dce-066f260f5409.png">

Reply:
Makes sense! Added the alias